src/sentry/spans/buffer.py

```diff
@@ -306,6 +307,29 @@ def _group_by_parent(self, spans: Sequence[Span]) -> dict[tuple[str, str], list[
 
         return trees
 
+    def get_stored_segments(self) -> int:
```

Comment:
The return value of this is no longer needed and we use this just to record metrics. Let's rename the function accordingly.